.travis.yml

```diff
@@ -0,0 +1,20 @@
+sudo: false
+language: cpp
+
+matrix:
+  include:
+    - os: linux
+      dist: trusty
+      compiler: gcc
+    - os: osx
+      osx_image: xcode9.4
+      compiler: clang
+
+before_install:
+  - eval "${MATRIX_EVAL}"
+
+script:
+  - cmake --version
+  - cmake -H. -Bbuild -DCMAKE_BUILD_TYPE=Release -DABSL_RUN_TESTS=ON -DABSL_USE_GOOGLETEST_HEAD=ON
+  - cmake --build build --target all
+  - cmake --build build --target test -- CTEST_OUTPUT_ON_FAILURE=1
```

Comment:
we use set TEST_OUTPUT_ON_FAILURE=1 earlier but pass it in a way which looks like a commandline flag here.  I like the first way better, let's try to be as consistent as possible with the scripts in between CI systems